web/js/helpers/requests.js

```diff
@@ -103,20 +103,6 @@
     return sendXHR('DELETE', `${server}/archive/${id}`);
   }
 
-  function saveConnection(connection, sessionId) {
-    return sendXHR('POST', `${server}/saveConnection/`, JSON.stringify({
-      connection,
-      sessionId
-    }), 'application/json');
-  }
-
-  function deleteConnection(connection, sessionId) {
-    return sendXHR('POST', `${server}/deleteConnection/`, JSON.stringify({
-      connection,
-      sessionId
-    }), 'application/json');
-  }
-
```

Comment:
Please delete `deleteConnection` and `saveConnection` from the `Request` object below.

Reply:
thxs, sorry